### PR TITLE
Improve Waypoint: Override Styles in Subsequent CSS #5466

### PR DESCRIPTION
**Pull request by wakqasahmed**
provides clarity about the concept of 'listed order' while specifying multiple classes

### CODE REVIEW

File: seed/challenges/01-front-end-development-certification/html5-and-css.json [hunk 1]
```diff
@@ -3539,7 +3539,8 @@
         "Create an additional CSS class called <code>blue-text</code> that gives an element the color blue. Make sure it's below your <code>pink-text</code> class declaration.",
         "Apply the <code>blue-text</code> class to your <code>h1</code> element in addition to your <code>pink-text</code> class, and let's see which one wins.",
         "Applying multiple class attributes to a HTML element is done with a space between them like this:",
-        "<code>class=\"class1 class2\"</code>"
+        "<code>class=\"class1 class2\"</code>",
+        "Note: It doesn't matter, which order the classes are listed in."
```
Comment (ltegman) [inline seed/challenges/01-front-end-development-certification/html5-and-css.json]: There should not be a comma in the middle of this sentence.

File: seed/challenges/01-front-end-development-certification/html5-and-css.json [hunk 2]
```diff
@@ -3575,7 +3576,8 @@
         "Crea una clase CSS adicional llamada <code>texto-azul</code> que le de a un elemento el color azul. Asegúrate de que está debajo de tu declaración de la clase <code>pink-text</code>. ",
         "Aplica la clase <code>blue-text</code> a tu elemento <code>h1</code> además de tu clase <code>pink-text</code> y veamos cual gana.",
         "La aplicación de múltiples atributos de clase a un elemento HTML se hace usando espacios entre ellos así:",
-        "<code>class=\"clase1 clase2\"</code>"
+        "<code>class=\"clase1 clase2\"</code>",
```
Comment (ltegman) [inline seed/challenges/01-front-end-development-certification/html5-and-css.json]: Please correct these to `\"class1 class2\"` while you're at it :smile:

**Comment by ltegman** [comment]
After you make the two requested changes please `git add . && git commit --amend && git push -f` to amend this commit and force push the changes to your remote branch. Thanks!

**Comment by wakqasahmed** [comment]
Done with said changes..

**Comment by ltegman** [comment]
Thanks!

**Pull request merged**